### PR TITLE
Reporting

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: grails-app/services/de/laser/ReportingGlobalService.groovy [hunk 1]
```diff
@@ -211,37 +231,43 @@ class ReportingGlobalService {
...
+                    if (prefix in [ BaseConfig.KEY_COSTITEM ]) {
+                        result.list = idList ? CostItem.executeQuery('select ci from CostItem ci where ci.id in (:idList) order by ci.costTitle', [idList: idList]) : []
+                        result.tmpl = TMPL_PATH + 'details/costItem'
+                    }
+                    else if (prefix in [ BaseConfig.KEY_LICENSE ]) {
                         result.list = idList ? License.executeQuery('select l from License l where l.id in (:idList) order by l.sortableReference, l.reference', [idList: idList]) : []
                         result.tmpl = TMPL_PATH + 'details/license'
                     }
-                    else if (prefix in ['licensor', 'org', 'member', 'consortium', 'provider', 'agency']) {
+                    else if (prefix in [ 'licensor', 'org', 'member', 'consortium', 'provider', 'agency' ]) {
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/ReportingGlobalService.groovy:259]: Leftover or intention that strings are hard-coded and not referred as constants?

**Review by croqueGrec09 — COMMENTED** [review]
one question

File: grails-app/services/de/laser/ReportingGlobalService.groovy (hunk 1 above)
Comment (klodav) [reply grails-app/services/de/laser/ReportingGlobalService.groovy:259]: Intention

Comment (croqueGrec09) [reply grails-app/services/de/laser/ReportingGlobalService.groovy:259]: Alright ...

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**